common/network-common/pom.xml

```diff
@@ -22,7 +22,7 @@
   <parent>
     <groupId>org.apache.spark</groupId>
     <artifactId>spark-parent_2.12</artifactId>
-    <version>3.3.0-SNAPSHOT</version>
+    <version>spark-755999</version>
```

Comment:
Is this necessary for testing? obv. just need to revert at some point. Not sure what the build will do with this

Reply:
Oh... I was running tests in parallel in another tab and it turns out that `./dev/run-tests` script modifies the `pom.xml` files. I'll redo the commits.

Reply:
Actually I don't know for sure what modified them - it could've been the test run script.
Fixed.